code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -105,6 +105,10 @@ This file's folder contains:
 			R.UnlinkSelf()
 			R.emagged = 1
 			R << "<span class='boldwarning'>You have been desynced from your master AI. In addition, your onboard camera is no longer active and your safeties have been disabled.</span>"
+		else if(isAI(S))
+			var/mob/living/silicon/ai/A = S
+			for(var/R in A.connected_robots)
```

Comment:
# :rotating_light:  :warning:  WARNING!! :warning: :rotating_light: 
The AI's `connected_robots` list is **not accurate**. You either need to check each borg and use it's ai var to confirm that the AI is indeed its master, or change the AI's borg list to remove desynced borgs (this is intended as a way of hiding the emagging process) to ensure that it is always correct.